api-report/container-runtime.api.md

```diff
@@ -328,6 +330,12 @@ export interface IContainerRuntimeOptions {
     loadSequenceNumberVerification?: "close" | "log" | "bypass";
     // (undocumented)
     summaryOptions?: ISummaryRuntimeOptions;
+    useDataStoreAliasing?: boolean;
+}
+
+// @public
+export interface IDataStore extends IFluidRouter {
+    trySetAlias(alias: string): Promise<boolean>;
```

Comment:
I do not think it's actionable yet, so take it mostly as an FYI.
I've been poking at offline story, and I think as much as possible we should avoid async APIs and instead explore other ways of tracking such progress and resolving conflicts. See https://microsoft-my.sharepoint-df.com/:w:/p/vladsud/Ebqk9UJUDVZCqRQWz8V0ZrcBnjAwpDMlu8WKVFqJMsDTRg?e=T498xA for some details.
Sharing mostly from POV to start discussion and see what the next steps on this path would be.